src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java

```diff
@@ -74,11 +101,123 @@ public String getName() {
         if (isWindows) {
             plainCek = decryptColumnEncryptionKeyWindows(masterKeyPath, encryptionAlgorithm,
                     encryptedColumnEncryptionKey);
-        } else {
+        } else if (null != keyStorePwd && null != keyStorePath) {
+            plainCek = decryptColumnEncryptionKeyNonWindows(masterKeyPath, encryptionAlgorithm, encryptedColumnEncryptionKey);

```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck
**Severity**: ERROR
**File**: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java L105

Line is longer than 80 characters (found 126).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1141273938>*